draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/SecondaryActions.tsx

```diff
@@ -51,33 +39,36 @@ const SecondaryActions = ({
   secondaryOverflowMenuItems,
   reversed = false,
 }: Props) => {
-  if (!secondaryActions) return null
+  if (!secondaryActions && !secondaryOverflowMenuItems) return null
 
-  const stuff = secondaryActions.map(a => {
-    if (isMenuGroupNotButton(a)) {
-      return (
-        <Menu
-          button={
-            <Button
-              secondary
-              label={a.label}
-              reversed={reversed}
-              icon={chevronDownIcon}
-              iconPosition="end"
-            />
-          }
-        >
-          <MenuContent>
-            {a.menuItems.map(menuItem => (
-              <MenuItem {...menuItem} />
-            ))}
-          </MenuContent>
-        </Menu>
-      )
-    } else {
-      return <Button secondary reversed={reversed} {...a} />
-    }
-  })
+  let stuff
+  if (secondaryActions) {
+    stuff = secondaryActions.map(a => {
+      if (isMenuGroupNotButton(a)) {
+        return (
+          <Menu
+            button={
+              <Button
+                secondary
+                label={a.label}
+                reversed={reversed}
+                icon={chevronDownIcon}
+                iconPosition="end"
+              />
+            }
+          >
+            <MenuContent>
+              {a.menuItems.map(menuItem => (
+                <MenuItem {...menuItem} />
+              ))}
+            </MenuContent>
+          </Menu>
+        )
+      } else {
+        return <Button secondary reversed={reversed} {...a} />
+      }
+    })
+  }
 
   let toolbarItems = stuff
```

Comment:
Why not toolbarItems from the start?

Reply:
Ah that `stuff` was not meant to be there, cheers